epregressions/diffs/ci_compare_script.py

```diff
@@ -191,6 +191,18 @@ def main_function(file_name, base_dir, mod_dir, base_sha, mod_sha, make_public,
             has_small_diffs = True
             print_message("Table small diffs.")
 
+    if entry.idf_diffs and (entry.idf_diffs.diff_type != TextDifferences.EQUAL):
+        has_small_diffs = True
+        print_message("IDF diffs.")
+
+    if entry.stdout_diffs and (entry.stdout_diffs.diff_type != TextDifferences.EQUAL):
+        has_small_diffs = True
+        print_message("StdOut diffs.")
+
+    if entry.stderr_diffs and (entry.stderr_diffs.diff_type != TextDifferences.EQUAL):
+        has_small_diffs = True
+        print_message("StdErr diffs.")
+
```

Comment:
CI will now respond to diffs in the IDF file itself, standard output, and standard error.